lib/getters/productGetters.ts

```diff
@@ -161,6 +161,26 @@ const getSegregatedOptions = (product: ProductCustom) => {
 const validateAddToCart = (product: ProductCustom): boolean =>
   isProductVariationsSelected(product) && Boolean(product.fulfillmentMethod)
 
+const getItemsLeft = (
+  product: ProductCustom,
+  productLocationInventoryData: Object,
+  fulfillmentOptionValue: string
+): number => {
+  const allVariantSelected = isProductVariationsSelected(product)
+  const qtyLeft = { value: 0 }
```

Comment:
we can make it a number datatype as well !

Reply:
Type added.